File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.js
function solution(babbling) {
    let answer = 0;
    for(let s of babbling){
        if(s.includes("aya")){
            s = s.replace("aya", " ");
        }
        if(s.includes("ye")){
            s = s.replace("ye", " ");
        }
        if(s.includes("woo")){
            s = s.replace("woo", " ");
        }
        if(s.includes("ma")){
            s = s.replace("ma", " ");
        }
        if (!s.trim()) answer++;
    }
    return answer;
}

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.java
class Solution {
    public int solution(String[] babbling) {
        int answer = 0;
        for(String s : babbling){
            if(s.contains("aya")){
                s = s.replace("aya", " ");
            }
            if(s.contains("ye")){
                s = s.replace("ye", " ");
            }
            if(s.contains("woo")){
                s = s.replace("woo", " ");
            }
            if(s.contains("ma")){
                s = s.replace("ma", " ");
            }
            if (s.trim().isEmpty()) answer++;
        }
        return answer;
    }
}